src/cmds/manifest/manifest.ts

```diff
@@ -190,9 +212,13 @@ export async function manifest(argv: YargsArgs): Promise<void> {
   }
 
   if (isInvalid) {
-    throw new Error(`Manifest Error: package.json validation failed, please see above errors.`);
+    throw new Error(
+      `Manifest Error: package.json validation failed, please see above errors.`,
```

Comment:
should we still use backtick or `'` if a string has no embedded expressions?

Reply:
I'm inclined to permit it if the lint config permits it, which it does. I think it's configurable. I'm not sure what we should prefer 🤔 

Captured here: https://github.com/MetaMask/eslint-config/issues/174